specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/webapplicationfirewall.json

```diff
@@ -306,6 +306,22 @@
           },
           "description": "A collection of references to application gateways."
         },
+        "httpListeners": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "./applicationGateway.json#/definitions/ApplicationGatewayHttpListener"
+          },
+          "description": "A collection of references to application gateway httplisteners."
+        },
+        "pathBasedRules": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "./applicationGateway.json#/definitions/ApplicationGatewayPathRule"
```

Comment:
If this is a list of references it should be of type `./network.json#/definitions/SubResource`